snuba/admin/clickhouse/predefined_querylog_queries.py

```diff
@@ -115,25 +115,25 @@ class QueryDurationForReferrerByProject(QuerylogQuery):
 
 
 class BeforeAfterBytesScannedComparison(QuerylogQuery):
-    """Given a certain time that abuse started on a certain cluster, specify a time range before and after the abuse (recommend 30 minutes). This will show the referrers with the largest change in bytes_scanned. In the example, the abuse started on Dec 7. at 21:54:50."""
+    """Given a certain time that abuse started on a certain cluster, specify a time range before and after the abuse (recommend 30 minutes). This will show the referrers with the largest change in bytes_scanned."""
 
     sql = """
     SELECT referrer, before_scanned, after_scanned, (after_scanned - ifNull(before_scanned, 0)) as diff, (diff / if(equals(after_scanned, 0), 1, after_scanned))  * 100 as pct_diff
     FROM
     (
         SELECT referrer, sum(arrayReduce('sum', clickhouse_queries.bytes_scanned)) as after_scanned
         FROM querylog_local
-        WHERE timestamp >= toDateTime('2022-12-07T21:55:00')
-        AND timestamp <= toDateTime('2022-12-07T22:20:00')
-        AND dataset IN ('events', 'discover')
+        WHERE timestamp >= toDateTime('<after_scanned_duration_start>')
+        AND timestamp <= toDateTime('<after_scanned_duration_end>')
+        AND dataset IN (<dataset>)
```

Comment:
Okay, originally I wanted to leave the dataset labels unquoted because I think the user might want to have multiple datasets, and if the label itself is quoted, they have the change the template itself. For instance, if the label is
```
dataset IN ('<dataset>')
```
They will not be able to do
```
dataset IN ('events', 'discover')
```
But I agree it's probably better to be consistent, so I updated the pr